http/task_service.go

```diff
@@ -379,7 +379,7 @@ func (h *TaskHandler) createBootstrapTaskAuthorizationIfNotExists(ctx context.Co
 		return nil, nil
 	}
 
-	prog, err := lang.Compile(t.Flux, time.Now())
+	prog, err := lang.Compile(nil, t.Flux, time.Now())
```

Comment:
This seems weird and I can't find the corresponding change on flux master.

Reply:
It's in `lang/compiler.go` line 87 (at the top in the files tab in the Flux PR)

Reply:
I see it now. I commented there.